tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -183,6 +174,8 @@
         - install-os
         - configure-os
         - run-os
+        - libvirt
+        - nova
```

Comment:
This does not seems to use the nova-compute-extraconfig created above.

Reply:
done